c2rust-analyze/src/rewrite/ty.rs

```diff
@@ -325,19 +326,38 @@ impl<'a, 'tcx> intravisit::Visitor<'tcx> for HirTyVisitor<'a, 'tcx> {
     fn nested_visit_map(&mut self) -> Self::Map {
         self.acx.tcx().hir()
     }
+
+    fn visit_stmt(&mut self, s: &'tcx hir::Stmt<'tcx>) {
+        match s.kind {
+            // A local with a user type annotation
+            StmtKind::Local(hir_local) if hir_local.ty.is_some() => {
```

Comment:
For consistency, this should use the qualified path `hir::StmtKind::Local` (and remove the `use hir::StmtKind` above).  Since this module is juggling a mix of HIR and `TyCtxt` representations, which often have overlapping names, I always use the fully qualified path `hir::Foo` or `ty::Foo` for clarity.

Reply:
3bc6f339